src/interceptors/fetch/index.ts

```diff
@@ -45,13 +46,18 @@ export class FetchInterceptor extends Interceptor<HttpRequestEventMap> {
         typeof input === 'string' &&
         typeof location !== 'undefined' &&
         !canParseUrl(input)
-          ? new URL(
-              input,
-              location.href
-            )
+          ? new URL(input, location.href)
           : input
 
       const request = new Request(resolvedInput, init)
+
+      /**
+       * @note Set the raw request only if a Request instance was provided to fetch.
+       */
+      if (input instanceof Request) {
```

Comment:
There is no point of exposing references to strings/URLs as you can grab them at `request.url` already.